src/docs/product/accounts/auth-tokens/index.mdx

```diff
@@ -0,0 +1,80 @@
+
+Different kinds of Auth Tokens contain access rights for different things. Under the hood, all Auth Tokens are tied to [Permissions & Scopes](/api/permissions/), which define all the things the Auth Token grants access for.
+
+Ideally, an Auth Token is only used once. This has the benefit that if the place you stored the Auth Token in is compromised, you can revoke this Auth Token without impacting any other place.
+
+## What Types of Auth Tokens Are There in Sentry?
+
+There are three key types of Auth Tokens in Sentry:
+
+- [Organization Auth Tokens](#organization-auth-tokens):
+  These tokens are bound to an Organization, and have access to all Projects within that Organization. They have a limited set of permissions, and are designed to be used in CI environments as well as for Sentry CLI.
+
+* [User Auth Tokens](#user-auth-tokens):
+  These tokens are bound to a User, and have access to all Organizations and Projects that User has access to.
+
+* [Internal Integrations](#internal-integrations):
+  These tokens are bound to an Organization, and have access to all Projects within that Organization. They can be created with a custom set of permissions, and are designed to be used in cases where Organization Auth Tokens do not have sufficient access rights.
+
+## When Should I Use Which?
+
+For most scenarios, we recommend to use [Organization Auth Tokens](#organization-auth-tokens). They are designed to be used in CI environments, and have a limited set of permissions. This means that if the place you stored the Auth Token in is compromised, the attacker can only do limited damage.
+
+They are also designed to be easily usable, they will work for all regular CI-related tasks out of the box.
+
+[User Auth Tokens](#user-auth-tokens) should be used when interacting with the Sentry API on behalf of a user. For example, if you want to fetch all issues for a user, you would use a User Auth Token. You should avoid using User Auth Tokens for e.g. CI processes, as due to their User-bound nature, if the user who created the token is ever removed from the Organization, the token will stop working.
+
+[Internal Integrations](#internal-integrations) should be used when you need full API access (which the Organization Auth Tokens cannot grant), and you want to interact with the Sentry API on behalf of an Organization. For example, if you want to create a new Project, you would use an Internal Integration.
+
+## Organization Auth Tokens
+
+[Organization Auth Tokens](https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/) can be created in the Sentry UI under **Settings > Developer Settings > Auth Tokens**.
+
+![](org-auth-tokens-overview.png)
+
+Each Auth Token has a user-defined name (which you can also update at a later point). This name is only used for display purposes, and does not have any impact on the Auth Token itself - it helps to identify an Auth Token e.g. in the case you want to revoke it later.
```

Comment:
This says the name is user-defined, but that's not true unless you create via the UI, right?

```suggestion
Organization auth token names are generated for you unless you create the token through the Sentry UI. This name is only used for display purposes - it helps to identify an auth token in case you want to revoke it later. You can change the name for an organization auth token at [sentry.io](https://sentry.io) on the **Edit Auth Token** page under  **Settings > Developer Settings > Auth Tokens**.
```